app/models/concerns/findings/csv.rb

```diff
@@ -64,6 +66,16 @@ def to_csv_a corporate
 
   private
 
+    def supervisor_review
+      supervisors = review.review_user_assignments.select do |rua|
```

Comment:
Probalo, creo que puede ir te queda en una linea

```suggestion
review.review_user_assignments.map{|rua| rua.user.full_name if rua.supervisor?}.compact.join '-'
```

Reply:
Seba está bueno, pero se me pasa 120 caracteres y debes tratar de mantener los 80

Reply:
Fijate que lo podes hacer con un do
```suggestion
      review.review_user_assignments.map do 
           |rua| rua.user.full_name if rua.supervisor?
      end.compact.join '-'
```